integration/basic_test.go

```diff
@@ -195,3 +197,59 @@ func TestCollectionName(t *testing.T) {
 		assert.Contains(t, names, longCollectionName)
 	})
 }
+
+func TestDatabaseName(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Err", func(t *testing.T) {
+		ctx, collection := Setup(t)
+
+		dbName300 := strings.Repeat("a", 300)
```

Comment:
Well, if we want to test boundary conditions, we should test with length 65 (or, at least, with length 65 too).
E.g. two cases: length 64 should pass validation, and length 65 should fail.

Reply:
> Well, at my first PR in #844 I also created such border cases. But, if you create this test, FerretDB would pass and fail in MongoDB.
> 
> To record that difference, I'll create a dance diff test as I did it for #844 PR. It's also specified in the issue description.

I misunderstood you.
Let me reformulate to highlight what.
Both databases have 64 letters max limit.
So you suggested testing 65 and 64-length names.

Reply:
but do we need to add positive test?

Reply:
> but do we need to add a positive test? we have it everywhere.

I checked and was surprised, you know, 
Database names MongoDB has max length 63, FerretDB has max length 64 :dancing_women: 

It's going to be in dance tests.